Classic console - Additional Instance Actions - Export / Create instance from snapshot (#613)
neo4j/docs-aura: modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc

Restoring a snapshot requires you to confirm the action by typing RESTORE and selecting *Restore*.

> === Export

Reviewer: ```suggestion
=== Export and create
```

> From the more menu (...) next to an existing snapshot, you can *Export*.
> *Export* downloads the instance as a compressed `.backup` file allowing you to store a local copy and work on your data offline.

Reviewer: ```suggestion
From the more menu (...) next to an existing snapshot, you can:
* *Export* - Download the instance as a *_.backup_* file (for latest version) or *_.dump_* (for version 4.x), allowing you to store a local copy and work on your data offline.
* *Create instance from snapshot* - Create a new instance using the data from the snapshot.
```

> === Create instance from snapshot 
> 
> You can create an instance using the data from a snapshot, just like creating a clone of an instance.
> From the more (...) menu next to an existing snapshot select *Create instance from snapshot*.
> This opens a window where you can assign a name to the new instance.

Reviewer: ```suggestion
```

Restoring a snapshot requires you to confirm the action by typing RESTORE and selecting *Restore*.

=== Additional snapshot actions (Export and create)

The ellipses (*...*) button next to an existing snapshot provides two options: *Export* or *Create instance from snapshot*.

> ==== Export

Reviewer: ```suggestion

```
Try to avoid nesting down to this level

*Export* downloads the instance as a `.backup` file, allowing you to store a local copy and work on your data offline.

> For AuraDB instances (including AuraDB with the GDS plugin installed) backups are always exported as a `.backup` file.

Reviewer: Is the plugin available in the old console?

Author: The plugin is only available for the new console, good spot.

> For AuraDS instances, backups are exported as a `.tar` file because this format bundles both the backup data and the additional metadata required for Data Science.

Author: Done!

> ==== Create instance from snapshot 

Reviewer: ```suggestion
=== Create instance from snapshot 
```
